catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -369,18 +396,36 @@ public Metacard transform(InputStream input) throws IOException, CatalogTransfor
         return transform(input, null);
     }
 
+    protected MetacardType getExtendedMetacardType(String metacardContentType) {
```

Comment:
New helper methods. Combines attributes descriptors from all the content extractors, metadata extractors, and the base metacard type to create a new metacardtype instance.